bottlecap/Cargo.toml

```diff
@@ -19,6 +19,7 @@ dogstatsd = { git = "https://github.com/DataDog/libdatadog", branch = "main" }
 figment = { version = "0.10", default-features = false, features = ["yaml", "env"] }
 hyper = { version = "0.14", default-features = false, features = ["server"] }
 lazy_static = { version = "1.5", default-features = false }
+libc = "0.2"
```

Comment:
Can we use the same format of other lines and specify the default-features=false? Or the minimum we need?